sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.integration.service.ts

```diff
@@ -147,8 +148,22 @@ export class IER12IntegrationService extends SFTPIntegrationBase<void> {
       ierFileDetail.totalExpectedContribution = combineDecimalPlaces(
         ierRecord.studentAndSupportingUserContribution,
       );
+      ierFileDetail.dependantChildQuantity = ierRecord.dependantChildQuantity;
+      ierFileDetail.dependantChildInDaycareQuantity =
+        ierRecord.dependantChildInDaycareQuantity;
+      ierFileDetail.dependantInfantQuantity = ierRecord.dependantInfantQuantity;
+      ierFileDetail.dependantOtherQuantity =
+        ierRecord.dependantDeclaredOnTaxesQuantity;
+      ierFileDetail.dependantPostSecondaryQuantity =
+        ierRecord.dependantPostSecondaryQuantity;
       ierFileDetail.totalDependantQuantity = ierRecord.totalEligibleDependents;
       ierFileDetail.familyMembersQuantity = ierRecord.familySize;
+      ierFileDetail.parent1Flag = this.convertToYNFlag(
+        ierRecord.numberOfParents > 0,
+      );
+      ierFileDetail.parent2Flag = this.convertToYNFlag(
+        ierRecord.numberOfParents === 2,
+      );
```

Comment:
To refresh my mind, if there are 2 parents then, both parent 1 and 2 flags will be `Y` right?

Reply:
@ann-aot yes as per the analysis output.